setup.py

```diff
@@ -39,7 +40,7 @@
     entry_points={
         'console_scripts': [
             'superflore-gen-ebuilds = superflore.generators.ebuild:main',
-            'superflore-gen-meta-pkgs = superflore.generators.bitbake:main',
+            'superflore-gen-oe-recipes = superflore.generators.bitbake:main',
```

Comment:
this might be a PR to chunk out? This doesn't depend on other PR's.

Reply:
Sounds good; done in https://github.com/ros-infrastructure/superflore/pull/178!